src/bootstrap/src/lib.rs

```diff
@@ -1790,8 +1791,7 @@ Executed at: {executed_at}"#,
                 .set_accessed(t!(metadata.accessed()))
                 .set_modified(t!(metadata.modified()));
 
-            let dst_file = t!(fs::File::open(dst));
-            t!(dst_file.set_times(file_times));
+            t!(set_file_times(dst, file_times));
```

Comment:
(Locally I added a comment for why this is even needed, due to the set file permissions changing file access times at least on linux).